app/controllers/crops_controller.rb

```diff
@@ -231,4 +168,28 @@ def crop_params
                                                       :_destroy,
                                                       :id])
   end
+
+  def filename
+    "Growstuff-Crops-#{Time.zone.now.to_s(:number)}.csv"
+  end
+
+  def crop_json_fields
```

Comment:
I tend to use rabl in other projects; perhaps a short term would be to push it to an ERB JSON template:

`format.json render :partial => "users/show.json"`